src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.builds

```diff
@@ -10,6 +10,12 @@
       <TargetGroup>net462</TargetGroup>
     </Project>
     <Project Include="System.ComponentModel.TypeConverter.csproj">
+      <TargetGroup>net463</TargetGroup>
+    </Project>
+    <Project Include="System.ComponentModel.TypeConverter.csproj">
```

Comment:
We should eliminate the netstandard1.0 and netstandard1.5 build configurations as those versions will come from harvesting the last stable package.